scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -25,19 +25,26 @@ class ThemeAggregator {
   /**
    * Aggregates theme assets.
    * @param {string} theme - The theme to override the default theme. Used for visual regression testing.
+   * @param {object} object.config - Config to override the terra-theme.config.js config.
+   * @param {object} object.aggregateDefaultThemeAsScopedTheme - Bool indicating if aggregate themes should generate the default theme as a root theme or a scoped theme.
    * @returns {string|null} - The output path of the aggregated theme file. Null if not generated.
    */
-  static aggregate(theme) {
+  static aggregate(theme, { config, aggregateDefaultThemeAsScopedTheme } = { config: undefined, aggregateDefaultThemeAsScopedTheme: false }) {
     let themeConfig = {};
-    const defaultConfig = path.resolve(process.cwd(), CONFIG);
-    if (fs.existsSync(defaultConfig)) {
-      // eslint-disable-next-line global-require, import/no-dynamic-require
-      themeConfig = require(defaultConfig);
+    if (config) {
+      themeConfig = config;
+    } else {
+      const defaultConfig = path.resolve(process.cwd(), CONFIG);
+      if (fs.existsSync(defaultConfig)) {
+        // eslint-disable-next-line global-require, import/no-dynamic-require
+        themeConfig = require(defaultConfig);
+      }
     }
 
     const themeAssets = ThemeAggregator.aggregateThemes({
       ...themeConfig,
       ...theme && { theme },
```

Comment:
Could we check for `theme` and `aggregateDefaultThemeAsScopedTheme` here to remove the added logic below in ThemeAggregator.aggregateThemes?

Reply:
Nevermind. I see below we switch between root vs scope aggregation